data/test/markdown_single_sections/not_existing_section.md

```diff
@@ -0,0 +1,8 @@
+## synonym:animal
```

Comment:
what's the difference to the `empty_section` one?

Reply:
this file contains a not existing section, e.g. `entity`. the issue mentioned that we just ignored unknown sections before and we just added all values of that unknown section to the section that was listed before.

Reply:
ah, `entity` looked to valid to me 🙈